### ISSUE TITLE
Pin environment to Julia versions

### ISSUE THREAD

**Issue by bryanparthum** [bryanparthum/schfc-paper#3]
@lrennels, can we pin the Julia version (and any others that we need) in this repository in the same way that we did the others? Some folks have been trying to replicate our estimates and getting the warning about Julia versions and different estimates.

**Comment by bryanparthum** [comment]
Let's only worry about this for `MimiGIVE`, not the IWG stuff (just to keep it simple). And should we remove the `MimiGIVE/Manifest.toml` from this repository as well?

**Comment by bryanparthum** [comment]
let's do the same thing we did [here](https://github.com/USEPA/scghg/pull/4)

**Comment by lrennels** [comment]
@bryanparthum sounds good, I can take this on this week

**Comment by lrennels** [comment]
@bryanparthum looking through things our README currently says the following

## Requirements
1. *Julia* is free and available for download [here](https://julialang.org/). Estimation of the SC-HFCs for the MimiGIVE and MimiIWG damage modules was performed on Julia 1.8. While newer versions of *Julia* are compatible with all the code (e.g. 1.9), the random number generators were updated and results might not be identical due to random differences in the random parameters underlying the Monte Carlo runs of MimiGIVE and MimiIWG. Install *Julia* and ensure that it can be invoked (ran) from where the replication repository is to be cloned ("in your path"). 

---
So I dug back into commits and realized that MimiIWG and MimiGIVE (all primary analysis) were run on Julia 1.6 by you, but I think MimiIWG_FAIRv162, just that small sensitivity analysis, was run by me on Julia 1.8. 

I will update the Manifests and README to reflect that as we did with scghg.

**Comment by bryanparthum** [comment]
Sounds good. My local is `1.6.3`, but I'll have to fire up a DMAP instance to confirm the version there (I believe it is also 1.6, but not 100%)

**Comment by lrennels** [comment]
Sounds good, I can on the readme be even more specific to `1.6.3` but I don't think patch (the versioning goes major.minor.patch) is likely to change the random number generator so it's probably not necessary.

What I ended up doing was going back to the commit before I updated all the Manifests and pulling the exact Manifests that were there before I did that commit. I think that should work nicely, but I can also just recreate them since I know we were on Julia 1.6.3 and 1.8.5 respectively.

I can test by running on my machine, but other testers would be good too, at least for the main results (aka no need to redo the sensitivity to MimiIWG + MimiFAIR)

**moving more details of what I did to the PR for discussion/documentation**
